scripts/promote

```diff
@@ -2,20 +2,41 @@
 
 set -ex
 
+# test channel
+CHANNEL=latest-ci-test
+
+# get RC version from npm before we promote it to latest!
+CLI_RC_VERSION=$(npm show sfdx-cli --json | jq -r '.["dist-tags"] | .["latest-rc"]')
+echo "version to promote: $CLI_RC_VERSION"
+
+# figure out the sha from the release
+SHA=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/salesforcecli/sfdx-cli/tags | jq -cr '.[] | select(.name=='\"v$CLI_RC_VERSION\"') | .commit | .sha')
+echo "found matching sha of ${SHA}"
+
+SHA_SHORT=$(git rev-parse --short=7 ${SHA})
+echo "will use sha of ${SHA_SHORT}"
+
+echo "will run command"
+echo "yarn promote --version $CLI_RC_VERSION --sha $SHA_SHORT --channel $CHANNEL --max-age 300 --macos --win"
+
+# promote it
+yarn promote --version $CLI_RC_VERSION --sha $SHA_SHORT --channel $CHANNEL --max-age 300 --macos --win
+
 # get updated channel files from last commit
-for channelfile in `git diff --name-only HEAD~1 HEAD | grep "^channels/*"`
-do
-  read -r _dir channel <<< $(echo $channelfile | tr "/" " ")
-  read -r version sha <<< $(head -n 1 $channelfile | tr "-" " ")
-
-  if [ -z "$sha" ]
-  then
-    echo "SHA is empty, promoting tag v$version to channel $channel"
-    git checkout v$version
-    sha=$(git rev-parse --short HEAD)
-    yarn promote --version $version --sha $sha --channel $channel --max-age 300 --macos --win
-  else
-    echo "Promoting $version-$sha to channel $channel"
-    yarn promote --version $version --sha $sha --channel $channel --max-age 300 --macos --win
-  fi
-done
+
+# for channelfile in `git diff --name-only HEAD~1 HEAD | grep "^channels/*"`
```

Comment:
If we go with this approach, which I think is better, then remove this and the channels folder.